introduction.md

```diff
@@ -1,2 +1,4 @@
 On my way to becoming a GitHub Master
-https://www.codeschool.com/paths/ruby
\ No newline at end of file
+https://www.codeschool.com/paths/ruby
+* https://www.codeschool.com/paths/ruby
+* https://www.codeschool.com/paths/javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?